tests/data/transformer-module/01-module-with-errors/expected_errors.json

```diff
@@ -1,22 +1,10 @@
 [
-  {
```

Comment:
While we no longer have this error in the transformer-module cases, it's coming from the new semantic validation cases. In order to have it flagged here I'd have to introduce yet another test folder which I think is unnecessary